r4/src/main/java/org/opencds/cqf/r4/providers/PlanDefinitionApplyProvider.java

```diff
@@ -179,6 +179,7 @@ private void resolveDefinition(Session session, PlanDefinition.PlanDefinitionAct
                         .buildContained(plan)
                         .addAction(new RequestGroupActionBuilder()
                             .buildResource(new Reference("#" + plan.getId()))
+                            .buildRelatedAction(action.getRelatedAction())
```

Comment:
Is it possible for `action.getRelatedAction()` to return null? I haven't tested out any examples yet, but it might be good to have a check for null in the `RequestGroupActionBuilder.buildRelatedAction` method

Reply:
`getRelatedAction` returns `List<PlanDefinition.PlanDefinitionActionRelatedActionComponent`. The docs make no reference to returning null so I would assume if there are no relatedActions it returns an empty list

Reply:
Just checked the actual source code for the `PlanDefinitionActionComponent` class and it does indeed return an empty list if `relatedAction` is null